Add tests. Apply new formatting.

ekateluv opened the pull request.

Commit:
Add tests. Apply new formatting.

webconn reviewed (commented):
Нужна зависимость от `python3-pytest` в `debian/control`. Возможно, из-за этого тесты не запускаются на CI.
И версию надо будет увеличить на единичку в патче

```
...
dh_auto_test -O--buildsystem=pybuild
I: pybuild base:184: cd '/<<BUILDDIR>>/modbus-utils-rpc-1.0.0~exp~feature+48116+tests~1~g6fbb15c/.pybuild/pythonX.Y_3.5/build'; python3.5 -m unittest discover -v 

----------------------------------------------------------------------
Ran 0 tests in 0.000s

OK
...
```

Commit:
Fix debian/control,  changelog

Fix build-depends

Add python3-six build-depends

Fix pytest version (3.0.6)

Fix debian/control python-mock

Fix test

webconn commented on debian/control:
```diff
@@ -3,10 +3,10 @@ Maintainer: Ekaterina Volkova <ekaterina.volkova@wirenboard.ru>
 Package: python3-modbus-utils-rpc
 Architecture: all
-Depends: python3, ${misc:Depends}, python3-umodbus (>=1.0.4-1+wb1), python3-paho-mqtt (<1.5.0), python3-mqttrpc(>=1.1.2)
+Depends: ${misc:Depends}
```
Отсюда не надо было убирать зависимости, они не берутся автоматически из build-depends

Commit:
Fix debian/control, add test main() function
